utils/utils_test.go

```diff
@@ -32,3 +35,20 @@ func TestMakeUniqueValue(t *testing.T) {
 	assert.Equal(t, "GetCPUlocalhost9090", testUnique2)
 	assert.Equal(t, "GetCPU128.97.26.119090", testUnique3)
 }
+
+func TestInitializeChannel(t *testing.T) {
+	sigs := InitializeChannel()
+	go func() {
+		time.Sleep(time.Millisecond * 100) // Wait a bit before sending the signal
+		if err := syscall.Kill(syscall.Getpid(), syscall.SIGINT); err != nil {
+			fmt.Printf("Failed to kill process: %v\n", err)
+		}
+	}()
+
+	select {
+	case <-sigs:
+		// Test passed, signal received
+	case <-time.After(time.Second):
+		t.Error("Expected to receive signal, but did not")
+	}
```

Comment:
What do you think adding assertion here to check having a signal.

Reply:
@xellos00 
Added